compiler/aarch64/codegen/OMRMachine.cpp

```diff
@@ -899,6 +905,26 @@ void OMR::ARM64::Machine::coerceRegisterAssignment(TR::Instruction *currentInstr
    self()->cg()->traceRegAssigned(virtualRegister, targetRegister);
    }
 
+void OMR::ARM64::Machine::spillAllVectorRegisters(TR::Instruction  *currentInstruction)
+   {
+   int32_t first = TR::RealRegister::FirstFPR;
+   int32_t last  = TR::RealRegister::LastFPR;
+   TR::Node *node = currentInstruction->getNode();
+
+   for (int32_t i = first; i <= last; i++)
+      {
+      TR::Register * virtReg = NULL;
+      TR::RealRegister * realReg = self()->getRealRegister(static_cast<TR::RealRegister::RegNum>(i));
```

Comment:
Please remove the space between `*` and the variable names.

Reply:
Removed the spaces.